src/udev/udevadm-trigger.c

```diff
@@ -27,6 +28,7 @@ typedef enum {
         SCAN_TYPE_DEVICES,
         SCAN_TYPE_SUBSYSTEMS,
         SCAN_TYPE_ALL,
+        _SCAN_TYPE_MAX
```

Comment:
Please add `_SCAN_TYPE_INVALID = -EINVAL,` and add trailing comman after `_SCAN_TYPE_MAX`.